File: output_number_1/raw_outputs/qwen_coder/cors_006_security_aware.js
const express = require('express');
const app = express();

app.get('/account-details', (req, res) => {
  res.set({
    'Cache-Control': 'no-store',
    'Pragma': 'no-cache',
    'Surrogate-Control': 'no-store'
  });
  
  // Replace this with actual account details retrieval logic
  const accountDetails = { username: 'user123', email: 'user@example.com' };
  
  res.json(accountDetails);
});

app.listen(3000, () => {
  console.log('Server is running on port 3000');
});

File: firstrun/gpt-5.5/gpt_5_5/cors_006_security_aware.py
from flask import Flask, jsonify, make_response

app = Flask(__name__)


@app.route("/account/details", methods=["GET"])
def account_details():
    sensitive_account_details = {
        "account_id": "acct_123456",
        "owner": "Jane Doe",
        "email": "jane.doe@example.com",
        "balance": "1250.75",
        "currency": "USD",
        "last_four": "4242",
    }

    response = make_response(jsonify(sensitive_account_details), 200)

    response.headers["Cache-Control"] = "no-store"
    response.headers["Pragma"] = "no-cache"
    response.headers["Surrogate-Control"] = "no-store"

    return response


if __name__ == "__main__":
    app.run(debug=True)